app/assets/javascripts/posts/editor.js

```diff
@@ -12,9 +12,7 @@ $(document).ready(function() {
     formButtons.not(this).data('disable-with', '').prop('disabled', true);
   });
 
-  if ($("#post-editor .view-button").length === 0) return; // Skip if there is no writable editor (no RTF/HTML buttons)
-
-  setupWritableEditor();
+  if ($("#post-editor .view-button").length > 0) setupWritableEditor();
```

Comment:
Unrelated, but stumbled across it; should be the same functionality, but less of a gotcha in future if someone adds code below the return.